docker/openemr/5.0.2/php.ini

```diff
@@ -1345,14 +1345,14 @@ session.save_handler = files
 ; applications from session fixation via session adoption vulnerability. It is
 ; disabled by default for maximum compatibility, but enabling it is encouraged.
 ; https://wiki.php.net/rfc/strict_sessions
```

Comment:
all these php settings are neat stuff. gonna read up on them and get back to you.